generators/server/templates/src/main/java/package/web/rest/ClientForwardController.java.ejs

```diff
@@ -23,22 +23,40 @@ import org.springframework.web.bind.annotation.GetMapping;
 
 @Controller
 public class ClientForwardController {
+    // File names with extension.
+    public static final String EXTENSION_PATTERN = "/{file:[^\\.]*}";
+
+    // Folders with trailing separator.
+    public static final String TRAILING_PATTERN = "/{traillingDir:.*}/";
```

Comment:
Is `traillingDir` an expression or can it be changed to `trailingDir`?

Reply:
typo
```suggestion
    public static final String TRAILING_PATTERN = "/{trailingDir:.*}/";
```